packages/framework/framework.css

```diff
@@ -1,21 +1 @@
-/*
-
-CSS magic will be available here soon!
-
-               |))    |))
- .             |  )) /   ))
- \\   ^ ^      |    /      ))
-  \\(((  )))   |   /        ))
-   / G    )))  |  /        ))
-  |o  _)   ))) | /       )))
-   --' |     ))`/      )))
-    ___|              )))
-   / __\             ))))`()))
-  /\@   /             `(())))
-  \/   /  /`_______/\   \  ))))
-       | |          \ \  |  )))
-       | |           | | |   )))
-       |_@           |_|_@    ))
-      /_/           /_/_/
-
-*/
```

Comment:
why did you remove the unicorn? :(

Reply:
Sorry about it! :'-(

Reply:
My god, such a beautiful unicorn was deleted!

Reply:
It always can be restored without affecting the functionality of the component! ;)